www/include/configuration/configObject/meta_service/DB-Func.php

```diff
@@ -170,18 +171,23 @@ function multipleMetaServiceInDB($metas = array(), $nbrDup = array())
                     $query = "SELECT DISTINCT contact_id FROM meta_contact WHERE meta_id = '" . $key . "'";
                     $dbResult = $pearDB->query($query);
                     while ($Contact = $dbResult->fetch()) {
-                        $query = "INSERT INTO meta_contact VALUES ('" .
-                            $maxId["MAX(meta_id)"] . "', '" . $Contact["contact_id"] . "')";
-                        $pearDB->query($query);
+                        $query = "INSERT INTO meta_contact VALUES (:max_mc_id, :contact_id)";
+                        $statement = $pearDB->prepare($query);
+                        $statement->bindValue(':max_mc_id', (int) $maxId["MAX(meta_id)"], \PDO::PARAM_INT);
+                        $statement->bindValue(':contact_id', (int) $Contact["contact_id"], \PDO::PARAM_INT);
```

Comment:
```suggestion
                        $statement->bindValue(':contact_id', (int) $contact["contact_id"], \PDO::PARAM_INT);
```
variable declaration should also be changed l.173

Reply:
@jeremyjaouen done :) !